statefun-end-to-end-tests/statefun-sanity-itcase/pom.xml

```diff
@@ -29,6 +29,7 @@ under the License.
 
     <properties>
         <testcontainers.version>1.12.5</testcontainers.version>
+        <flink.version>1.10.0</flink.version>
```

Comment:
I think we don't need to define this here. Parent POM already defines `flink.version` so we should inherit that.

Reply:
Ah scratch that 😅 It's only defined in `statefun-flink`.
nevermind then :)